config/docker/data/kmod_kci_bookworm.patch

```diff
@@ -0,0 +1,35 @@
+diff -Naur kmod-30+20221128/debian/changelog kmod-30+20221128-kernelci/debian/changelog
```

Comment:
Has this patch been proposed to Debian? It won't help for bookworm, but that would help once we switch to the next stable.

Reply:
This is specific to our needs in ChromiumOS, i dont know why, but they use zlib... (while they invented zstd for example). Not sure it can be upstreamed.